packages/zoe/src/contracts/callSpread/types.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @typedef {'long' | 'short'} PositionKind
+ */
+
+/**
+ * @typedef {Object} PayoffHandler
+ * @property {() => void} schedulePayoffs
+ * @property {(positionKind: PositionKind) => Promise<Payment>} makeOptionInvitation
```

Comment:
Let's move this out into it's own definition so we can use in line 29 of payoffHandler